analyticsdataserver/tests/utils.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.auth.models import User
```

Comment:
This should be the same as the tests that were in `analyticsdataserver/tests.py`.

Reply:
these are helpful notes

another method I like is to break refactors into a separate commit so you can look at just the interesting changes or just the boring changes